airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/buffered_stream_consumer/BufferedStreamConsumer.java

```diff
@@ -66,6 +66,17 @@
  * <p>
  * All other message types are ignored.
  * </p>
+ *
+ * <p>
+ * Throughout the lifecycle of the consumer, records get promoted from buffered to flushed to
```

Comment:
I think I got a little confused by this message. Is this copy overloading `record` with `message`?
```suggestion
 * Throughout the lifecycle of the consumer, messages get promoted from buffered to flushed to
```